modules/apps/info/info-api/src/main/java/com/liferay/info/field/InfoField.java

```diff
@@ -47,12 +45,15 @@ public InfoField(
 
 		this(
 			builder(
-				infoFieldType, name
+			).infoFieldType(
+				infoFieldType
+			).name(
+				name
 			).labelInfoLocalizedValue(
 				labelInfoLocalizedValue
 			).localizable(
 				localizable
-			));
+			)._builder());
```

Comment:
In AMImageQueryBuilder this method is called done()
I would rename it to be consistent.

Reply:
I'll rename it, but I'll keep it private because its only purpose is to be used in those deprecated constructors.